Update "User title" (#7936)
ppy/osu-wiki: wiki/Community/User_title/en.md

*For the list of people who have attained special user titles, see [Users with unique titles](/wiki/People/Users_with_unique_titles).*

![Screenshot of Ephemeral's profile information](img/dev.png "The \"osu!team\" title can be seen near the top of Ephemeral's profile.")

A **user title** is a short label on a user's profile page set by the [osu! team](/wiki/People/The_Team). Most titles indicate membership of a [user group](/wiki/People/The_Team#user-groups), but some users have earned [more unique titles](/wiki/People/Users_with_unique_titles) for specific achievements or contributions.

> The colour of a user title is the same as the user's primary group membership badge. If the user has no group badge, the title is white.

Reviewer: do you mean that the two are decoupled now or that the previous text is correct?

These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.

Post count titles were discontinued with the release of the new osu! website.

## Trivia

- User titles are referred to as "rank titles" throughout the game's code. This is a leftover name from when osu! was built on top of [phpBB](https://www.phpbb.com/).

> - As of March 2021, If a user is eligible to multiple user titles, they would be displayed together on their profile, separated by a slash, instead of only displaying their most recent user title. <!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->

Reviewer: ```suggestion
- As of March 2021, if a user is eligible for multiple user titles, they are displayed together on their profile separated by a slash, instead of only displaying their most recent user title.<!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->
```

> ::: Infobox

Reviewer: put this above the paragraph you want it to appear to the side of

- User titles are referred to as "rank titles" throughout the game's code. This is a leftover name from when osu! was built on top of [phpBB](https://www.phpbb.com/).

> - As of March 2021, if a user is eligible for multiple user titles, they are displayed together on their profile separated by a slash, instead of only displaying their most recent user title.<!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->

Reviewer 2: this isn't really true either. lots of people are eligible for multiple titles but don't have them

also, the wording makes it sound like it's a feature of titles themselves, when really this is just a single title assigned manually ... i feel it should say somethign with this idea
```suggestion
- As of March 2021, users earning multiple titles are usually assigned a title that joins all of them together with slashes. Previously, it was more common to display only the most significant or most recent title.
```
